src/Altinn.Apps/KubernetesWrapper/KubernetesWrapper.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
we should use dotnet core 2.2 as this is what altinn studio will be updated to, and the platform sln are also on dotnet core 2.2. Rule of thumb: all new dotnet projects should be dotnet core 2.2

Reply:
Fixed.